astra/src/test/java/com/slack/astra/server/AstraTest.java

```diff
@@ -275,14 +274,6 @@ public void testDistributedQueryOneIndexerOneQueryNode() throws Exception {
             indexerMeterRegistry);
     indexer.serviceManager.awaitHealthy(DEFAULT_START_STOP_DURATION);
 
-    await()
```

Comment:
we wait for the service to be healthy. That should be enough?

We don't pre-register  any fields so this counter is 0